probables/blooms/bloom.py

```diff
@@ -348,7 +346,7 @@ def close(self) -> None:
             self.__file_pointer.close()
             self.__file_pointer = None
 
-    def __load(self, filepath: Union[str, Path], hash_function: Union[HashFuncT, None] = None):
+    def load(self, filepath: Union[str, Path], hash_function: Union[HashFuncT, None] = None):
```

Comment:
I would prefer if this wasn't a top level function. Leaving it as `__load` works and has users go through the current initialization process.

Reply:
perhaps renaming to `__load_from_file` or something similar?

Reply:
I [use it directly](https://github.com/KOLANICH-libs/ProbSetAbs.py/blob/master/ProbSetAbs/backends/PyProbables.py) on precreated filter objects. Double underscore doesn't really prevent usage, just makes it even more fragile. Is is here not to make private properties but to avoid accidental overwrite when used with mixins. Usually double underscore props should be avoided.

Reply:
So you set up a Bloom filter and then re-initialize it from file? You can do that directly in the initialization (ie initialize from file). Is there a reason you do it that way? 

Besides curiosity, this pr looks good.

Reply:
I can do it and can do it, but I feel like it'd be a bit more code and can be slower. Just the first library I had wrapped had the API in the fashion "create then load".  And pyprobables internally [uses the same model](https://github.com/barrust/pyprobables/blob/86c6692342847b1928810fe2fa695abfe19d4124/probables/blooms/bloom.py#L132-L133), in the factory methods, first the structure is created and then it is populated.